### PR TITLE
fix: Updates region to include tasks: datastore_batch_upsert

### PR DESCRIPTION
**Pull request by grant**
We need to include `task1` and `task2` in this sample for it to make sense.
b/111318350

### CODE REVIEW

**Comment by grant** [comment]
How do I force Kokoro to run?
This change shouldn't break the tests.

**Comment by ace-n** [comment]
cc @engelke

**Comment by grant** [comment]
Trying to push a file to get the tests to run

File: vision/cloud-client/document_text/env/.Python [hunk 1]
```diff
@@ -0,0 +1 @@
+/System/Library/Frameworks/Python.framework/Versions/2.7/Python
```
Comment (engelke) [inline vision/cloud-client/document_text/env/.Python]: Did you mean to include these env files?

Comment (grant) [reply vision/cloud-client/document_text/env/.Python]: That file is outdated and creds are revoked.

**Pull request merged**